modules/apps/transfer/keeper/grpc_query_test.go

```diff
@@ -181,16 +182,21 @@ func (suite *KeeperTestSuite) TestQueryDenomHash() {
 	testCases := []struct {
 		msg      string
 		malleate func()
-		expPass  bool
+		expErr   error
 	}{
+		{
+			"success",
+			func() {},
+			nil,
+		},
 		{
 			"invalid trace",
 			func() {
 				req = &types.QueryDenomHashRequest{
-					Trace: "transfer/channelToA/transfer/",
+					Trace: "transfer%%/channel-1/transfer/channel-1/uatom",
```

Comment:
This test was not triggering the "invalid trace" error but the "denom not found one". Thanks @crodriguezvega for the help!!

Reply:
Is the %% what triggers invalid trace here?

Reply:
To my understanding yes, but this is all @crodriguezvega!

Reply:
Yes, correct. The `%%` triggers invalid trace error.